### ISSUE TITLE
Unable to install OS with `Accessibility Tools` checked

### ISSUE THREAD

**Issue by st22nestrel** [CachyOS/CachyOS-Handheld#73]
Hi,

I tried to install CachyOS handheld edition numerous times on my ROG Ally z1e and each time it failed with this error (whole pacman log [pacman_log.txt](https://github.com/user-attachments/files/22877257/pacman_log.txt)):
```
2025-10-11 - 21:40:51 [6]: [PYTHON JOB]: "error: failed to commit transaction (conflicting files)" 
2025-10-11 - 21:40:51 [6]: [PYTHON JOB]: "orca: /usr/lib/systemd/user/orca.service exists in filesystem (owned by steamos-manager)" 
2025-10-11 - 21:40:51 [6]: [PYTHON JOB]: "Errors occurred, no packages were upgraded." 
```

It took me a while to figure that this error is related to package `orca` from `Accessibility Tools` which I do not need to install (but I wanted to). After unchecking `Accessibility Tools` I was able to install CachyOS without errors.

I guess this problem happens on any hardware when you have `Accessibility Tools`, or more precise `orca` service checked (I cannot say for sure as I unchecked whole `Accessibility Tools` section). Could you fix this problem, maybe add some warning to  checkbox section or error message that will hint user to uncheck `orca` service, so they do not have to bug their head for a while as I did?

**Comment by 1Naim** [comment]
Hi, this is fixed by https://github.com/CachyOS/CachyOS-PKGBUILDS/commit/a5b3defad35953974d93a6929eacba9f28cdc486. Prior to orca 49.0, it didn't bundle its own service file so steamos-manager had to create their own for its purposes. I've removed the one from steamos-manager with this commit, and this should be fixed now. 

Thanks!
